ivanti_security_controls/plugin.spec.yaml

```
@@ -389,6 +389,182 @@ types:
       description: The vendor-defined severity of the security risk or issue that this patch corrects
       type: string
       required: true
+  next:
+    href:
+      title: "Href"
+      type: string
+      description: "Href"
+      required: false
+  links:
+    next:
+      title: "Next"
+      type: next
+      description: "Next"
+      required: false
+  links_self:
+    self:
+      title: "Self"
+      type: next
+      description: "Self"
+      required: false
+  vulnerability:
+    bulletinId:
+      title: "Bulletinid"
+      type: string
+      description: "Bulletinid"
+      required: false
+    cve:
+      title: "CVE"
+      type: "[]string"
+      description: "CVE"
+      required: false
+    id:
+      title: "Id"
```

If this is the Ivanti Patch ID, let's call it that so it's clear

```suggestion
      title: "Patch ID"
```